### ISSUE TITLE
AttributeError: 'time.struct_time' object has no attribute 'tm_gtoff'

### ISSUE THREAD

**Issue by skiesx** [stanvanrooy/instauto#85]
Hi, i found a bug.

My code 

```
import random
import os
os.environ['TZ']= 'Europe/Kiev'
from time import sleep

from instauto.api.client import ApiClient
from instauto.api.actions import search as se

if os.path.isfile('./.instauto.save'):
    client = ApiClient.initiate_from_file('./.instauto.save')
else:
    client = ApiClient(user_name=os.environ.get("INSTAUTO_USER") or "username", password=os.environ.get("INSTAUTO_PASS") or "pass")
    client.login()
    client.save_to_disk('./.instauto.save')

s = se.Username("danyilmazurenko", 1)
resp = client.search_username(s).json()
user_id = resp['users'][0]['pk']
print(user_id)
```

and I got an error
```

Traceback (most recent call last):
  File "instauto_test.py", line 20, in <module>
    s = se.Username("danyilmazurenko", 1)
  File "...../lib/python3.6/site-packages/instauto/api/actions/structs/search.py", line 13, in __init__
    self.timezone_offset = str(time.localtime().tm_gtoff)
AttributeError: 'time.struct_time' object has no attribute 'tm_gtoff'
```

**Comment by stanvanrooy** [comment]
Fixed in #82

**Comment by skiesx** [comment]
> 
> 
> Fixed in #82

@stanvanrooy when will you can upload a new version to pypi?
